Tools/Scripts/build-and-analyze

```diff
@@ -170,6 +184,10 @@ def main(args):
     ]
 
     if args.dry_run:
+        env_vars_to_show = ['ANALYZER_EXEC', 'ANALYZER_FLAGS', 'ANALYZER_OUTPUT']
+        filtered_env = { key: value for key, value in os.environ.items() if key in env_vars_to_show }
+        for key, value in filtered_env.items():
```

Comment:
You could combine the 3 lines into one expression and skip constructing a new dict:
`print('\n'.join(f'{key}="{value}"' for key, value in os.environ.items() if key in env_vars_to_show)`

Reply:
Thanks!  Will fix the commit log typo, and I used your one-liner above!

Tested using `--dry-run` (though I also did some actual build & analyze runs):

```
$ ./Tools/Scripts/build-and-analyze --sdkroot macosx --output-dir /private/tmp/analyzer --configuration release --disable-checkers unix --enable-checkers unix.Malloc --dry-run
PATH_TO_SCAN_BUILD="scan-build"
ANALYZER_FLAGS="-analyzer-disable-checker unix -analyzer-checker optin.cplusplus.UninitializedObject,optin.cplusplus.VirtualCall,unix.Malloc"
ANALYZER_OUTPUT="/private/tmp/analyzer"

.../OpenSource/Tools/Scripts/set-webkit-configuration --release

make analyze SDKROOT=macosx

.../OpenSource/Tools/Scripts/generate-static-analysis-archive --count --output-root /private/tmp/analyzer --id-string WebKit Build
```